calamari_ocr/ocr/datasets/input_dataset.py

```diff
@@ -202,7 +202,7 @@ def text_generator(self) -> Generator[str, None, None]:
     def generator(self, epochs=1, text_only=False) -> Generator[Tuple[np.array, List[str], Any], None, None]:
         self.check_initialized()
         for epoch in range(epochs):
-            if self.mode == DataSetMode.TRAIN:
+            if self.mode == DataSetMode.TRAIN:          #only train here, pred and eval are covered by else block 
```

Comment:
space, and maybe move comment into the block (new line)

Reply:
done